include/rawtoaces/acesrender.h

```diff
@@ -72,7 +73,7 @@ class AcesRender
     const AcesRender &operator=( const AcesRender &acesrender );
 
```

Comment:
Suggestion is to move impl. bit of LibRawAces into .cpp
so lines 17-24 replace with `class LibRawAces;` and move all this into .cpp instead